src/main/java/frc/trigon/robot/subsystems/elevator/ElevatorConstants.java

```diff
@@ -0,0 +1,183 @@
+package frc.trigon.robot.subsystems.elevator;
+
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.signals.*;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import frc.trigon.robot.constants.RobotConstants;
+import org.trigon.hardware.phoenix6.cancoder.CANcoderEncoder;
+import org.trigon.hardware.phoenix6.cancoder.CANcoderSignal;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXMotor;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXSignal;
+import org.trigon.hardware.simulation.ElevatorSimulation;
+import org.trigon.utilities.mechanisms.ElevatorMechanism2d;
+
+public class ElevatorConstants {
+    private static final int
+            MASTER_MOTOR_ID = 12,
+            FOLLOWER_MOTOR_ID = 13,
+            ENCODER_ID = 12;
+    private static final String
+            MASTER_MOTOR_NAME = "ElevatorMasterMotor",
+            FOLLOWER_MOTOR_NAME = "ElevatorFollowerMotor",
+            ENCODER_NAME = "ElevatorEncoder";
+    static final TalonFXMotor
+            MASTER_MOTOR = new TalonFXMotor(MASTER_MOTOR_ID, MASTER_MOTOR_NAME),
+            FOLLOWER_MOTOR = new TalonFXMotor(FOLLOWER_MOTOR_ID, FOLLOWER_MOTOR_NAME);
+    static final CANcoderEncoder ENCODER = new CANcoderEncoder(ENCODER_ID, ENCODER_NAME, RobotConstants.CANIVORE_NAME);
```

Comment:
1. you don't have an encoder, you should rely on limit switches to reset 
2. this is not connected to a CANivore